pytest.ini

```diff
@@ -1,4 +1,4 @@
 [pytest]
-addopts = -ra -q --tb=short --numprocesses auto --import-mode=importlib
+addopts = -ra -q --tb=short --import-mode=importlib
```

Comment:
issue: This changes how tests are run in CI, I don't think we should make this change, and if we should, let's do it in a separate PR where it can be discussed.

FWIW, I personally use direnv with this line:

```
export PYTEST_ADDOPTS="-vvvs -n0 --kafka-bootstrap-servers=localhost:9092 --timeout=300"
```

Reply:
It's something that I forgot to remove 😄 because otherwise the debugger of pycharm wasn't working. Good I'm going to try with the Env!